tools/src/vendoring/devmenu/steps/Task.ts

```diff
@@ -35,7 +35,7 @@ export abstract class Task {
    * @returns the override working directory for task.
    */
   protected overrideWorkingDirectory(): string | undefined {
-    return;
+    return undefined;
```

Comment:
Can we not require to explicitly return `undefined`? Also side note, I'm not a big fan of using `undefined` anywhere. We should use `null` instead, it clearly indicates that the function intentionally returned this value, we can't say that when `undefined` is returned from the function that is meant to return some value.